core/src/main/java/nl/nn/adapterframework/webcontrol/api/Base.java

```diff
@@ -186,10 +188,12 @@ protected String resolveStringFromMap(Map<String, List<InputPart>> inputDataMap,
 		return result;
 	}
 
-	protected <T> T resolveTypeFromMap(Map<String, List<InputPart>> inputDataMap, String key, Class<T> clazz, T defaultValue) throws ApiException {
+	@SuppressWarnings("unchecked")
+	protected <T> T resolveTypeFromMap(List<Attachment> inputDataMap, String key, Class<T> clazz, T defaultValue) throws ApiException {
 		try {
-			if(inputDataMap.get(key) != null) {
-				return inputDataMap.get(key).get(0).getBody(clazz, null);
+			String att = this.getAttributeValue(key, inputDataMap).orElse(null);
+			if (null != att) {
+				return (T) att;
```

Comment:
You cannot just cast a String to an object

Reply:
i already fixed it :)